examples/cube.html

```diff
@@ -28,6 +31,29 @@
     fsButton.addEventListener('click', function() {
       scene.enterVR();
     });
+
+    if (window.top !== window.self) {
+      fsButton.parentNode.removeChild(fsButton);
+
+      window.addEventListener('message', function (e) {
+        if (e.data && e.data.type === 'fullscreen') {
+          if (e.data.data === 'enter') {
+            scene.setStereoRenderer();
+          } else if (e.data.data === 'exit') {
+            scene.setMonoRenderer();
+          }
+        }
+      });
+
+      window.top.postMessage({type: 'ready'}, '*');
+
+      document.getElementById('cube').addEventListener('click', function (e) {
+        var href = e.target.getAttribute('href');
+        if (href) {
+          window.top.postMessage({type: 'navigate', data: {url: href}}, '*');
+        }
+      });
+    }
```

Comment:
Will work on moving the above code into either vr-scene, the add-on, or into it's own library.    Once we get deeper into integrating the add-on, we can make a decision then.